Source/BugsnagErrorReportApiClient.m

```diff
@@ -114,35 +95,56 @@ - (void)sendReports:(NSArray<BugsnagCrashReport *> *)reports
     }
 }
 
-@end
-
-@implementation BSGDelayOperation
```

Comment:
This was added in https://github.com/bugsnag/bugsnag-cocoa/commit/42944e0032c0b31011430ae7795c6df6f10de14b#diff-3ae0af9a6224b0b591a4a57c922e273e to avoid a case where several repeated notify calls or exceptions were either dropped or delivered twice. Is there comparable handling for this now elsewhere for the asynchronous delivery case?